### PR TITLE
Added Vulnerability check job

### PR DESCRIPTION
**Pull request by soumeh01**

### CODE REVIEW

File: .github/workflows/eventlist.yml [hunk 1]
```diff
@@ -22,14 +26,14 @@ jobs:
...
-        go: [1.17.x]
+        go: [1.19.3]
     name: 'Build'
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
 
-      - name: Install go 1.17
+      - name: Install go 1.19
```
Comment (JonatanAntoni) [inline .github/workflows/eventlist.yml:36]: Can we use the variable here? I.e. `Install go §{{ matrix.go }}`?
If not, I'd remove the version from the step name.

File: .github/workflows/eventlist.yml [hunk 2]
```diff
@@ -161,6 +165,28 @@ jobs:
           test ! -s build/format-check.out
         working-directory: ./tools/eventlist
 
+  vulnerability-check:
+    name: "Vulnerability check"
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        go: [1.19.3]
```
Comment (JonatanAntoni) [inline .github/workflows/eventlist.yml:173]: Hm, we repeat the Go version multiple times. But I guess the very same version should be used in all places. Can we factor this version into a central place?

**Review by JonatanAntoni — APPROVED** [review]

File: .github/workflows/eventlist.yml (hunk 1 above)
Comment (soumeh01) [reply .github/workflows/eventlist.yml]: [Addressed review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/c0f043365bfd7f76b3f4cb4e2f9e6146237363b7)

File: .github/workflows/eventlist.yml (hunk 2 above)
Comment (soumeh01) [reply .github/workflows/eventlist.yml]: [Addressed review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/c0f043365bfd7f76b3f4cb4e2f9e6146237363b7)

File: .github/workflows/eventlist.yml [hunk 3]
```diff
@@ -194,17 +197,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, windows-2019, macos-12]
-        go: [1.19.3]
     name: 'Test (${{ matrix.os }}, ${{ matrix.go }})'
```
Comment (JonatanAntoni) [inline .github/workflows/eventlist.yml:200]: `${{ matrix.go }}` is not set anymore

Comment (soumeh01) [reply .github/workflows/eventlist.yml]: Ahh My bad. Fixed it
[Address missing review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/d8b0aa11de4d1688de7f97c1bff7beb73562326f)

**Review by JonatanAntoni — APPROVED** [review]

**Pull request merged**